pkg/ibmvpc/examples/out/explain_out/LBToIksNode_all_vpcs_explain_debug.txt

```diff
@@ -1,5 +1,5 @@
-Explaining connectivity from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca to 192.168.4.4 (iks-node[192.168.4.4]) within ky-test-vpc
-========================================================================================================================================
+Explaining connectivity from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca[LoadBalancer] to 192.168.4.4 (iks-node[192.168.4.4]) within ky-test-vpc
```

Comment:
The input `src` is `crn:v1:staging:public:is:us-south:a/6527::vpc:a456`. This should be reflected in the header. So that the user knows what she queried about.

Reply:
please advise for header convention, (we have user input, nodeSet and list of nodes), I will fix the code accordingly

Reply:
see suggestion in next item